apps/artemis-web/components/hijack-info/hijack-info.tsx

```diff
@@ -116,7 +316,7 @@ class HijackInfoComponent extends Component<any, any> {
 
     const type = this.getEventType(this.props.hijackDataState["type"]);
     const asn = this.props.hijackDataState["hijack_as"];
-    const prefix = this.props.hijackDataState["prefix"];
+    // const prefix = this.props.hijackDataState["prefix"];
```

Comment:
why is this commented out?

Reply:
It was not being used

Reply:
and why is only the `asn` and `type` used? To check for GRIP fetching we need all type, asn and prefix to be actually present.

Reply:
You are very right. However, this logic is included in   async fetchGrip(hijackDataState) {
 function.

The data are stored in hijackDataState. Prefix var was redundant

Reply:
I see, now it is clear. Resolving this pending comment.